src/witness/witness.ml

```diff
@@ -442,6 +442,36 @@ struct
         in
         (module TaskResult:WitnessTaskResult)
       )
+    | Termination -> (* TODO: implement this properly*)
```

Comment:
What should be changed here?

Reply:
There is nothing to be done here, as correctness witnesses are not need for termination.